packages/cacti-copm-core/src/main/typescript/lib/types.ts

```diff
@@ -15,7 +16,8 @@ export type CopmContractNames = {
 };
 
 export type RemoteNetworkConfig = {
-  channelName: string;
+  channelName: string; // fabric-specific
+  chaincode: string; // fabric-specific
```

Comment:
Please rename this to `chaincodeName`.

Reply:
Probably `networkName` instead of `network` in the below line 21?

Reply:
Hi @jenniferlianne I'm not sure about the purpose of this `type RemoteNetworkConfig`, but even the channelName and corda party endpoints could also be different for every remote call for the same network, as there can be multiple channels (as discussed on discord).

Its okay if you can just open an issue to track this.

Reply:
As discussed with @VRamakrishna 2 weeks ago, the naming, which is consistent with a similar structure in the fabric implementation submitted 3 months ago, reflects the data structures in the weaver test network json configuration files. While named 'network', they seem to talk about organizations, not networks.   While I may be wrong, I expect this information will not change on the fly for a given organization.  Can rename to RemoteOrganizationConfig.

Reply:
Lets it be as it is for now, just open an issue to track this, so that we can think about it more later, if we actually need restructuring.

Reply:
Too late - I updated the naming to be RemoteOrganization, as well as using 'organization' instead of 'network' for AssetAccounts in the OpenApi spec.  The code is more consistent now.  I opened an issue here for revisiting in the future: https://github.com/hyperledger-cacti/cacti/issues/3656